content/authors/divarvel.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Clément Delafargue"
+description = "Maintainer of Biscuit and biscuit-haskell"
+date = 2021-11-06T12:24:00+00:00
+updated = 2021-11-06T12:24:00+00:00
+draft = false
++++
+
+Maintainer of Biscuit and biscuit-haskell
+
+[@divarvel](https://github.com/divarvel)
```

Comment:
```suggestion
[@clementd](https://github.com/divarvel)
```

(Could you rename the file as well please?)

Reply:
sure